client/components/dt-room-creation.jsx

```diff
@@ -1,10 +1,12 @@
 import React from 'react';
 import createGameId from '../lib/create-game-id';
+import socketIOClient from 'socket.io-client';
 
 export default class RoomCreation extends React.Component {
   constructor(props) {
     super(props);
     this.state = {
+      endpoint: 'http://localhost:3001',
```

Comment:
Avoid hard-coding localhost into your urls, or else it won't work when your app is on a different domain name. Instead, you can substitute this for `'/'`